dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOptionTest.java

```diff
@@ -117,7 +117,7 @@ public void testConfigureAdditionalMavenRepositories_notToUseMavenCentral()
       throws ParseException {
     CommandLine commandLine =
         StaticLinkageCheckOption.readCommandLine(
-            new String[] {"-m", "https://repo.spring.io/milestone", "--no-maven-central"});
+            "-m", "https://repo.spring.io/milestone", "--no-maven-central");
```

Comment:
Look throughout the file for other places to do this instead of creating a `String[]` local variable.

Reply:
Removed the local variables.